candidates/index_config/works-candidate.json

```diff
@@ -805,6 +821,10 @@
               "base": {
                 "type": "text",
                 "analyzer": "base"
+              },
+              "normalized_keyword": {
```

Comment:
I understand the rationale behind this name, and perhaps it's just in the context of the above text->keyword changes, but I found it a bit odd that a field called keyword has type text.  Perhaps "normalised_whole_phrase" or something.

Reply:
Good point - was also a bit dissatisfied by this.

Reply:
I have a question; do we tend to adopt UK or US spelling for things on the BE? (normalised vs normalized)

Reply:
Good Q, generally US spellings: basically we want to avoid weird clashes like `color: ${colour}`, which sometimes means using UK spellings when dealing with e.g. public API parameters but usually means complying with US-centric languages/libraries/3rd party APIs. In this case ES talks a lot about `normalizer`s etc so it makes more sense to use US spelling.